### DISCUSSION TITLE
Custom SSR on Example - redirects to login

### DISCUSSION THREAD

**Discussion by ubresponsive** [gladly-team/next-firebase-auth#72] [Q&A] [answered]
I'm a newbie, so I might be doing something wrong but have the 'example' running, did a NPM install, added .env.local and all auths are working correctly except for the custom routing option (ssr-custom-routing.js) it redirects to login endpoint and comes up with a 404. The login endpoint exists. Any suggestions on what might be wrong?

**Comment by kmjennison** [comment, marked as answer]
This behavior is expected. The example is a little confusing, but if you check out [the source](https://github.com/gladly-team/next-firebase-auth/blob/main/example/pages/ssr-custom-routing.js#L60), it's meant to demonstrate custom routing based on auth state. In this case, it's redirecting to the login page when the user's email isn't verified.

**Reply by ubresponsive** [reply]
aha, that makes sense! always helps to read the source!
